SPECS/hdf5/hdf5.spec

```
@@ -0,0 +1,1054 @@
+BuildRequires: automake
+BuildRequires: libtool
+# Needed for mpi tests
+BuildRequires: openssh-clients
+BuildRequires: libaec-devel
+BuildRequires: gcc, gcc-c++
+
+
+%description
+HDF5 is a general purpose library and file format for storing scientific data.
+HDF5 can store two primary objects: datasets and groups. A dataset is
+essentially a multidimensional array of data elements, and a group is a
+structure for organizing objects in an HDF5 file. Using these two basic
+objects, one can create and store almost any kind of scientific data
+structure, such as images, arrays of vectors, and structured and unstructured
+grids. You can also mix and match them in HDF5 files according to your needs.
+
+%package devel
+Summary:        HDF5 development files
+Requires:       %{name}%{?_isa} = %{version}-%{release}
+Requires:       gcc-gfortran
+Requires:       libaec-devel%{?_isa}
+Requires:       zlib-devel%{?_isa}
+
+%description devel
+HDF5 development headers and libraries.
+
+%if %{with java}
+%package -n java-hdf5
+Summary: HDF5 java library
+Requires:  slf4j
+Obsoletes: jhdf5 < 3.3.2^
+
+%description -n java-hdf5
+HDF5 java library
+%endif
+
+%package static
+Summary: HDF5 static libraries
+Requires: %{name}-devel = %{version}-%{release}
+
+%description static
+HDF5 static libraries.
+
+
+%if %{with_mpich}
+%package mpich
+Summary: HDF5 mpich libraries
+BuildRequires: mpich-devel
+Provides: %{name}-mpich2 = %{version}-%{release}
+Obsoletes: %{name}-mpich2 < 1.8.11-4
+
+%description mpich
+HDF5 parallel mpich libraries
+
+
+%package mpich-devel
+Summary: HDF5 mpich development files
+Requires: %{name}-mpich%{?_isa} = %{version}-%{release}
+Requires: libaec-devel%{?_isa}
+Requires: zlib-devel%{?_isa}
+Requires: mpich-devel%{?_isa}
+Provides: %{name}-mpich2-devel = %{version}-%{release}
+Obsoletes: %{name}-mpich2-devel < 1.8.11-4
+
+%description mpich-devel
+HDF5 parallel mpich development files
+
+
+%package mpich-static
+Summary: HDF5 mpich static libraries
+Requires: %{name}-mpich-devel%{?_isa} = %{version}-%{release}
+Provides: %{name}-mpich2-static = %{version}-%{release}
+Obsoletes: %{name}-mpich2-static < 1.8.11-4
+
+%description mpich-static
+HDF5 parallel mpich static libraries
+%endif
+
+
+%if %{with_openmpi}
+%package openmpi
+Summary: HDF5 openmpi libraries
+BuildRequires: openmpi-devel
+BuildRequires: make
+
+%description openmpi
+HDF5 parallel openmpi libraries
+
+
+%package openmpi-devel
+Summary: HDF5 openmpi development files
+Requires: %{name}-openmpi%{?_isa} = %{version}-%{release}
+Requires: libaec-devel%{?_isa}
+Requires: zlib-devel%{?_isa}
+Requires: openmpi-devel%{?_isa}
+
+%description openmpi-devel
+HDF5 parallel openmpi development files
+
+
+%package openmpi-static
+Summary: HDF5 openmpi static libraries
+Requires: %{name}-openmpi-devel%{?_isa} = %{version}-%{release}
+
+%description openmpi-static
+HDF5 parallel openmpi static libraries
+%endif
+
+
+%prep
+%autosetup -n %{name}-%{version}%{?snaprel} -p1
```

Since we don't have `snaprel`, we can simplify:
```suggestion
%autosetup -p1
```